functions/update-dynamodb/handler.ts

```diff
@@ -0,0 +1,26 @@
+import { winstonLogger } from "../../shared/logger";
+import { DocumentClient } from "aws-sdk/clients/dynamodb";
+
+export const handle = async (): Promise<void> => {
+    const { PK, SK, DDB_TABLE_NAME } = process.env;
```

Comment:
This file still hives a lot of linting issues. Please take a look at the outcome of the linter from GitHub Actions here https://github.com/Drodevbar/per-feature-env/actions/runs/5576745328/jobs/10188482710 and correct this file to fix all linter issues